lib/core/services/people-content.service.spec.ts

```diff
@@ -94,6 +94,17 @@ describe('PeopleContentService', () => {
         });
     });
 
+    it('should call listPeople api with requested query params', (done) => {
+        const listPeopleSpy = spyOn(service.peopleApi, 'listPeople').and.returnValue(Promise.resolve(fakeEcmUserList));
+        const requestQueryParams: PeopleContentQueryRequestModel = { skipCount: 10, maxItems: 20, sorting: { orderBy: 'firstName', direction: 'asc' } };
+        const expectedValue = { skipCount: 10, maxItems: 20, orderBy: ['firstName ASC'] };
+
+        service.listPeople(requestQueryParams).subscribe(() => {
+            expect(listPeopleSpy).toHaveBeenCalledWith(expectedValue);
+            done();
+        });
```

Comment:
Could we try prefer writing tests using `async/await` instead of `done()`?
```suggestion
        await service.listPeople(requestQueryParams).toPromise();
        expect(listPeopleSpy).toHaveBeenCalledWith(expectedValue);
```

Reply:
updated